requirements.txt

```diff
@@ -27,8 +26,6 @@ pexpect==4.1.0
 pathlib2==2.1.0
 pickleshare==0.7.2
 simplegeneric==0.8.1
-path.py==8.2.1
-backports.shutil_get_terminal_size==1.0.0
```

Comment:
yeah, it's just the directory names in the pypi mirror that had to change. package names in requirements.txt stay the same.